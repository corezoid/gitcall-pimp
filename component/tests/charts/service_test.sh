#!/usr/bin/env bash
$CZ helm template | sed -n '/service\.yaml/,/Source\:/p'
