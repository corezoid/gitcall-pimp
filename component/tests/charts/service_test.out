# Source: gitcall_pimp_release/charts/gitcall_pimp/templates/service.yaml
apiVersion: v1
kind: Service
metadata:
  name: RELEASE-NAME-gitcall-pimp-service
  labels:
    release: "RELEASE-NAME"
    app: "gitcall"
    tier: "pimp"
spec:
  selector:
    release: "RELEASE-NAME"
    app: "gitcall"
    tier: "pimp"
  ports:
  - protocol: TCP
    port: 8080
    targetPort: 8080
---
# Source: gitcall_pimp_release/charts/gitcall_pimp/templates/deployment.yaml
